Source/Editor/Windows/SplashScreen.cpp

```diff
@@ -21,17 +21,11 @@ const Char* SplashScreenQuotes[] =
     TEXT("Reloading gun"),
     TEXT("Consuming your RAM"),
     TEXT("Burning your CPU"),
-    TEXT("#BetterThanUnity"),
     TEXT("Rendering buttons"),
     TEXT("Collecting crash data"),
-    TEXT("Downloading porn"),
 #if PLATFORM_WINDOWS
-    TEXT("Removing 'C:\\Windows\\'"),
 #elif PLATFORM_LINUX
-    TEXT("Time to switch to Windows?"),
-    TEXT("Installing Windows 10"),
```

Comment:
I think doubling down on the joke could work.

Something like `Running install_windows.bat` on Linux and `sudo apt-get linux` on Windows